Samples/src/Arithmetic/AdderExample.ipynb

```diff
@@ -235,15 +235,15 @@
    "outputs": [
     {
      "data": {
```

Comment:
is it better to commit the notebooks with no output?

Reply:
I could see either way, really. Including the output allows for the notebooks to be read on GitHub and nbviewer, but does make the files much larger, harder to maintain, etc.